src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantMandatoryQuestionsController.java

```diff
@@ -53,6 +52,9 @@ public ResponseEntity<InputStreamResource> exportDueDiligenceData(@PathVariable
 
         final ByteArrayOutputStream stream = grantMandatoryQuestionService.getDueDiligenceData(schemeId, isInternal);
         final String exportFileName = grantMandatoryQuestionService.generateExportFileName(schemeId, null);
+
+        submissionsService.updateLastRequiredChecksExportBySchemeIdAndStatus(schemeId);
```

Comment:
I would change the name of this to just updateLastRequiredChecksExportBySchemeId, since thats all we are doing from the method call point of view.